elements/pl-number-input/pl-number-input.py

```diff
@@ -323,12 +323,39 @@ def test(element_html, data):
         data['raw_submitted_answers'][name] = str(a_tru)
         data['partial_scores'][name] = {'score': 1, 'weight': weight}
     elif result == 'incorrect':
-        data['raw_submitted_answers'][name] = str(a_tru + (random.uniform(1, 10) * random.choice([-1, 1])))
         data['partial_scores'][name] = {'score': 0, 'weight': weight}
+        # Get method of comparison, with relabs as default
+        comparison = pl.get_string_attrib(element, 'comparison', 'relabs')
```

Comment:
The defaults in this and other calls to `pl.get_*_attrib` are now repeated several times throughout the file; I'm thinking we should have some dict of defaults that we can reference for consistency and DRYness. @mwest1066 thoughts?

Reply:
Maybe for now I can move the defaults to the head of the file as (fake) constant variables? Like CONST_COMPARISON_DEFAULT = 'relabs'

Reply:
Yep, that’s a good idea

Reply:
By that I just meant something like

```python
defaults = {
  'comparison': 'relabs'
}

# ...

comparison = pl.get_string_attrib(element, 'comparison', defaults['comparison'])
```

But yeah, this is functionally the same as just using `CONSTANTS` at the top 🙂

Reply:
For now I'm just using `CONSTANTS` at the top. If you decide to use something like `get_attribs(element)`, I can implement that.